File: Res/Searching/Linear Search/linearSearch.dart
int linearSearch(List<int> list, int target) {
  for (int i = 0; i < list.length; i++) {
    if (list[i] == target) {
      return i; // Target found, return the index
    }
  }
  return -1;
}

void main() {
  List<int> numbers = [10, 23, 45, 70, 11, 15];
  int target = 70;

  int result = linearSearch(numbers, target);

  if (result != -1) {
    print("Element found at index: $result");
  } else {
    print("Element nof found in the list.");
  }
}

File: Res/Searching/Linear Search/linearSearch.py
def linearSearch(arr, target):
    n = len(arr)
    for i in range(n):
        if arr[i] == target:
            return i
    return -1

numbers = [10, 23, 45, 70, 11, 15]
tar = 11
result = linearSearch(numbers, tar)
print(result)
if result != -1:
    print(f"Element is found at index: {result}")
else:
    print("Element is not found")
    